### ISSUE TITLE
Plugin started to crush after upgrading from 5.1.1 to 5.2.0 because of circular reference while trying to JSON.stringify actual and expecteed

### ISSUE THREAD

**Issue by Rikkun-1** [archfz/cypress-terminal-report#191]
After upgade from 5.1.1 to 5.2.0 I am getting following error.

```log
TypeError: The following error originated from your application code, not from Cypress.

  > Converting circular structure to JSON
    --> starting at object with constructor 'VueComponent'
    |     property '_scope' -> object with constructor 'EffectScope'
    |     property 'effects' -> object with constructor 'Array'
    |     index 0 -> object with constructor 'Watcher'
    --- property 'vm' closes the circle

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application
    at JSON.stringify (<anonymous>)
    at formatLogMessage (webpack:///./node_modules/cypress-terminal-report/src/collector/LogCollectCypressCommand.js:22:0)
    at $Cypress.eval (webpack:///./node_modules/cypress-terminal-report/src/collector/LogCollectCypressCommand.js:39:0)
```

I guess it's addresses this issue https://github.com/archfz/cypress-terminal-report/issues/184 and this line specifically https://github.com/archfz/cypress-terminal-report/commit/28a403bd2da7962c8bc348303a9d8b90992c5d51#diff-3978b46bd5deb2b0fa647bf73cdd404149578daec75579653a00827fcde24832R22-R23

**Comment by archfz** [comment]
Indeed. Contribution is welcome https://stackoverflow.com/questions/11616630/how-can-i-print-a-circular-structure-in-a-json-like-format

**Comment by archfz** [comment]
Fixed in 5.3.0
